src/Fantomas/CodePrinter.fs

```diff
@@ -5136,8 +5136,17 @@ and genSynBindingFunction
             (pref +> genOnelinerAttributes astContext ats)
 
     let afterLetKeyword =
+        let estimateRange = 
```

Comment:
This proves to be more challenging than I anticipated.
In essence, you want to use the start position of the entire top-level let binding (a range that includes the `let` keyword) and the start position of the function name `map`.
You know the `inline` keyword is somewhere in that range.

![image](https://user-images.githubusercontent.com/2621499/149285599-51cf8824-b40d-4a4d-a39f-dbdd551c28b8.png)

And then you end up with something like:
```fsharp
    let afterLetKeyword =
        let genInline (ctx: Context) =
            let letBindingRange : Range = failwith "information not present at this point"
            let estimateRange = ctx.MkRange letBindingRange.Start patRange.Start
            tokN estimateRange INLINE (!- "inline ")

        ifElse isMutable (!- "mutable ") sepNone
        +> ifElse isInline genInline sepNone
        +> opt sepSpace ao genAccess
```

The main problem is that inside that function you don't have any notion of the start of the prefix.
You could be inside of a member and then instead of `let` you started with `member`.
So, you would need to pass down the start position of the parent node of SynBinding.

As there are multiple places where `genSynBindingFunction` is eventually called this would be quite tricky.

An easier route would be if the range of the inline keyword was added to the Syntax tree in the compiler.
Because now we really have to hack our way around it to find an approximate range.
This sounds like a good candidate for SyntaxTrivia, https://github.com/dotnet/fsharp/issues/12418.